tests/mod-resolver/issue-5198/lib/explanation.txt

```diff
@@ -0,0 +1,16 @@
+This file is contained in the './lib' directory.
+
+The directory name './lib' conflicts with the './lib.rs' file name.
+
+'lib.rs' defines 3 external modules:
+
+    * mod a;
+    * mod b;
+    * mod c;
+
+Module resolution will fail if we look for './lib/a.rs' or './lib/a/mod.rs',
+so we should fall back to looking for './a.rs', which correctly finds the modlue that
```

Comment:
minor typo fyi

```suggestion
so we should fall back to looking for './a.rs', which correctly finds the module that
```

Reply:
Aww man, I'm sorry about that. I totally would have gone back and fixed this before you merged had you pointed this out.

Unrelated to the typo, but was the `explanation.txt` file helpful?

Reply:
Ha no worries. We've typos all over the place :laughing: 

And yes I think the text file with explanatory commentary is helpful in cases like these as it lowers the barrier required to grok especially when trying to eyeball runtime module loading behavior